docs/migrating_from_Vlc.DotNet.md

```diff
@@ -0,0 +1,85 @@
+
+Later, [Jérémy](https://github.com/jeremyVignelles) became the de factor maintainer of Vlc.DotNet as ZeBobo5 had less time to spend on the project.
+
+Mid 2018, VideoLAN started looking into .NET libvlc bindings. It was decided to create a new project, [LibVLCSharp](https://code.videolan.org/videolan/LibVLCSharp), instead of starting from Vlc.DotNet because we wanted to have a similar public API than [libvlcpp](https://code.videolan.org/videolan/libvlcpp) and design it to be crossplatform from scratch.
+
+## Platform support
+
+### Vlc.DotNet
+
+Vlc.DotNet is designed to work on Windows and Windows only. 
+
+> It provides support down to .NET Framework 3.5 (released in 2007) up to the latest versions of .NET Framework, as well as .NET Core 3+. It offers .NET Standard 1.3 builds as well.
+
+Vlc.DotNet provides View integrations for WPF as well as WinForms.
+
+### LibVLCSharp
+
+LibVLCSharp is designed to be fully crossplatform. Focus was initially mainly on iOS and Android, but desktop integration has become a primary focus as well.
+
+> 
+    Xamarin.Android
+    Xamarin.iOS
+    Xamarin.tvOS
+    Xamarin.Mac (Cocoa)
+    Windows Classic (WPF, WinForms, GTK)
+    Windows Universal (UWP for Desktop, mobile and Xbox)
+    Linux including desktop, server and Raspberry Pi (GTK)
+    Xamarin.Forms
+    Uno Platform (UWP, Android, iOS)
+    .NET Standard 1.1 and 2.0
+    .NET Framework 4.0 and later
+    .NET Core (including ASP.NET Core)
+    Unity3D
+        Windows Classic
+
+## API differences
+
+Both being based off of libvlc, the public APIs are quite similar. When in doubt, always inspect the bindings code to see which libvlc function it binds too.
+
+For loading LibVLC, unlike with Vlc.DotNet, you may just call `Core.Initialize()`. The path will be found automatically, provided you have installed the LibVLC Windows nuget (see section below).
+
+## LibVLC nuget vs VLC local installation
+
+To use any libvlc bindings, you need to first load libvlc and to do so, you need to know its location.
+
+Before, it was commonly done to use the local VLC app installation directory from your .NET app. It does contain all the necessary files and DLLs, after all.
+
+However it is not really a good idea for several reason. One, you may want to update your VLC app and not the libvlc build used by your .NET app. Making inter-dependent is not a good idea and there is no reason to. Another reason not to do that is you might want full control over your deployment (i.e. which libvlc build your client uses, and how to update).
+
+For those reasons and more, we created the [VideoLAN NuGet account](https://www.nuget.org/profiles/videolan). NuGet is the package manager for dotnet. Now you can just install any LibVLC version with just a single command. For LibVLC Windows version 3.0.10, type:
+```
+dotnet add package VideoLAN.LibVLC.Windows --version 3.0.10
+```
+
```

Comment:
Yes, this was written from a windows dev viewpoint since vlc.dotnet only supports windows. Though crossplatform support is clearly stated (but then the regular doc applies)